generated/agrifood/log.out

```diff
@@ -19,6 +19,5 @@ info    |    Loading local AutoRest extension '@autorest/bicep' (/home/runner/wo
 info    |    Loading AutoRest extension '@autorest/modelerfour' (*->4.23.5)
 warning | SkippedSemanticValidation | Semantic validation was skipped.
 warning | PreCheck/CheckDuplicateSchemas | Checking for duplicate schemas, this could take a (long) while.  Run with --verbose for more detail.
-info    | Resource type microsoft.agfoodplatform/farmbeats/extensions under path '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AgFoodPlatform/farmBeats/{farmBeatsResourceName}/extensions/{extensionId}' has no body defined.
```

Comment:
Was removing this logging an intentional change?

Reply:
This was caused by [allowing the resource body processor to fall back to the GET body](https://github.com/Azure/bicep-types-az/pull/805/files#diff-8c3d58262f34ad4e08ca80feda2fa4b3432558a1922526eef37fd22ed6950860R71). The intention of that change was to support cases where there is no PUT request defined, but I guess there were a few cases where there was a PUT request but it had no body.